packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -44,4 +44,5 @@
 
 .itemContainer {
   display: inline-block;
+  vertical-align: top;
```

Comment:
yes it is!!! any time you inline-block, you should strongly, strongly consider setting vertical alignment. Otherwise the browser will shuffle the div around trying to align its containing text with other text.